cmd/qat_plugin/dpdkdrv/dpdkdrv_test.go

```diff
@@ -405,6 +441,7 @@ func TestScan(t *testing.T) {
 				tt.maxDevNum,
 				tt.kernelVfDrivers,
 				tt.dpdkDriver,
+				tt.preferredAllocationPolicy,
```

Comment:
the `tcases` struct does not need to change because there's nothing to parameterize:
```suggestion
				"none",
```